backend/main.py

```
@@ -39,6 +41,9 @@
     version="1.0.0"
 )
 
+# データベースのテーブルを作成（起動時）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)